### PR TITLE
Add support for a chocolatey package

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

File: Build.proj [hunk 1]
```diff
@@ -170,6 +171,13 @@
     <Delete Files="@(TempCopied)" />
   </Target>
 
+  <Target Name="ChocoPackage" Condition="'$(OS)' != 'Unix' " DependsOnTargets="Stage;MakePackageDir"
+          Outputs="$(PackageDir)\ironpython.chocolatey.$(PackageVersion).nupkg">
+    <Exec Command="$(NuGet) install chocolatey -Version $(ChocolateyVersion) -OutputDirectory &quot;$(RootDir)\packages&quot;" />
```
Comment (slozier) [inline Build.proj]: Any way to do this via a `PackageReference` or something like that instead in invoking nuget directly?

Comment (slide) [reply Build.proj]: We currently don't do any restore on Build.proj, so we would have to do that prior to invoking the build.

Comment (slozier) [reply Build.proj]: Don't we have the following DependsOnTargets: `ChocoPackage` -> `Stage` -> `_PlatformStage` -> `RestoreReferences`?

Comment (slide) [reply Build.proj]: Yes, but that restores references on the solution using the builtin Restore target for solutions. Does that automatically work for a proj?

Comment (slozier) [reply Build.proj]: I have no idea. I'm fine with this as it is (assuming it works). We can always change it later.

Comment (slide) [reply Build.proj]: I think we would have to import some .targets file or something to get a default Restore target. I tried adding a restore call, but it said there was no Restore target for Build.proj.

**Pull request merged**